README.md

```diff
@@ -700,17 +700,21 @@ To set this up, perform the following steps:
 
 6. Select the `Build Settings` tab
 
-7. Click the `+` button on the toolbar and select `Add User-Defined Setting`
+7. Go to `Build Location -> Per-configuration Build Products Path -> Staging` and change `Staging` value from `$(BUILD_DIR)/$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME)` to `$(BUILD_DIR)/Release$(EFFECTIVE_PLATFORM_NAME)`
```

Comment:
Could you leave a short note here explaining why? (Similar to your PR description, with a link)